File: business/src/main/java/com/isystk/sample/common/dto/StripePaymentDto.java
package com.isystk.sample.common.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Stripe用の支払情報DTO
 */
public class StripePaymentDto implements Dto {

  @JsonProperty("client_secret")
  String clientSecret;

  public String getClientSecret() {
    return this.clientSecret;
  }

  @JsonProperty("client_secret")
  public void setClientSecret(String clientSecret) {
    this.clientSecret = clientSecret;
  }
}

File: business/src/main/kotlin/com/isystk/sample/common/dto/StripePaymentDto.kt
package com.isystk.sample.common.dto

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * Stripe用の支払情報DTO
 */
class StripePaymentDto : Dto {
    @set:JsonProperty("client_secret")
    @JsonProperty("client_secret")
    var clientSecret: String? = null
}